pkg/manager/task_clean.go

```diff
@@ -73,3 +77,36 @@ func (t *CleanTask) pathInStash() bool {
 	logger.Debugf("File %s is out from stash path", t.Scene.Path)
 	return false
 }
+
+func includeFile(file string, patterns []string) bool {
```

Comment:
Suggest moving this method, and `excludeFiles` from manager_tasks.go into a new `exclude.go` (or something like that) file so that the exclude list code is in the one place. It will also make it easier to refactor the common code in the two methods into helper methods.